cmd/katapult/error_utils.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/spf13/cobra"
+	"github.com/stretchr/testify/assert"
+)
+
+func executeTestCommand(t *testing.T, cmd *cobra.Command, errResult, wants, stderrResult string) {
+	stdout := &bytes.Buffer{}
+	stderr := &bytes.Buffer{}
+	cmd.SetOut(stdout)
+	cmd.SetErr(stderr)
+	err := cmd.Execute()
+	switch {
+	case err == nil:
+		// Ignore this.
+	case errResult != "":
+		assert.Equal(t, errResult, err.Error())
+		return
+	default:
+		t.Fatal(err)
+	}
```

Comment:
I think we can simplify this switch case to a single if condition, check my comment on `Test_newClient` for reference.